pkg/network/tracer/connection/kprobe/tracer.go

```diff
@@ -190,29 +181,27 @@ func (t *kprobeTracer) GetMap(name string) *ebpf.Map {
 	}
 }
 
-func (t *kprobeTracer) GetConnections(buffer []network.ConnectionStats, filter func(*network.ConnectionStats) bool) ([]network.ConnectionStats, error) {
+func (t *kprobeTracer) GetConnections(active, closed *network.Buffer, filter func(*network.ConnectionStats) bool) (int, int, error) {
```

Comment:
Do we need the separate count return values, couldn't you get the length of the connections slice returned from the buffer? Or even call `Buffer.Len()`?